tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -188,6 +188,12 @@ bool PVConstraint::liftedOnCVars(const ConstraintVariable &O,
 bool PVConstraint::isLt(const ConstraintVariable &Other, 
                         ProgramInfo &Info) const 
 {
+  if (isEmpty() || Other.isEmpty())
```

Comment:
You need to swap the order of the if-statements here.    You should test `A && B` before `A || B`.